install/jakartaee/bin/sig-test_se8.map

```diff
@@ -143,15 +143,15 @@ jakarta.resource=2.0_se8
 
 # JAXR 1.0 
 # (This is JSR-093)
-#jakarta.xml.registry=1.0_se8
+#javax.xml.registry=1.0_se8
```

Comment:
JAXR is pruned from EE 9 so probably should remove javax.xml.registry

Reply:
Should be removed now